Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
  * Allow qualified access to events from other contracts.
 
 Compiler Features:
+ * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode (especially from native binaries vs emscripten binaries)
```

Comment:
```suggestion
 * Code Generator: Fix not entirely deterministic order of functions in unoptimized Yul output. The choice of C++ compiler in some cases would result in different (but equivalent) bytecode, especially from native binaries vs emscripten binaries.
```

Also, shouldn't this be in the  `Bugfixes` section?

Reply:
#14355 seems to have a similar entry, but with an improved version of the text. Just noting this so we don't forget or overwrite that.

Reply:
> Also, shouldn't this be in the `Bugfixes` section?

Good catch. Of course it should :) Looks like I did not pay enough attention when I was moving it :)

> #14355 seems to have a similar entry, but with an improved version of the text. Just noting this so we don't forget or overwrite that.

Yeah, that's because it's based on this PR. No worries, one reason for keeping it as a draft is to avoid someone merging it accidentally without first merging this PR.